canonical/ubuntu.com
# add meganav yaml proof of concept

## Done

- Added a mega nav item that pulls data from a new YAML file

## QA

- Visit 
- See that the "Get Ubuntu" item is populated with items from the meganav.yaml that this PR adds - it is completely unstyled currently, this PR is the result of a spike to see whether it was possible to pull from a YAML file when the template is dynamically added to the page on user interaction.

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/4603
